docs/src/services/proxy/metrics.md

```diff
@@ -97,6 +97,12 @@ The proxy exposes the following metrics around sessions:
   The duration it took for a `filter`'s `write` implementation to execute.
   * The `filter` label is the name of the filter being executed.
 
+* `filter_int_counter{id, label, help, direction}`
```

Comment:
I think it would be good to have a section on Filter's generic metrics and how it gets generated, with examples of what label, help, direction etc could possibly be as well.